### PR TITLE
Feat new theme

### PR DESCRIPTION
**Pull request by krshdev**
closes #ISSUE-NO

#### What does this PR do?

Adds a new theme, navbar and sidebar

#### Description of the task to be completed?

- [x] Theme
- [x] Navbar
- [x] Sidebar 

#### How can this be manually tested?

- [x] Deployed to [gh-pages](https://krishguptadev.github.io/support/#/)

#### Any background context you want to provide?

We can then customize the theme [using the available options](https://docsify-darklight-theme.boopathikumar.me/#/configuration?id=default-themes-configuration)

#### Screenshots (if appropriate)

image

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: docs/_layout/navbar.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!-- markdownlint-disable-next-line -->
+- [Discord](http://discord.eddiehub.org)
+- [Twitter](https://twitter.com/eddiejaoude)
+- [Youtube](https://youtube.com/c/eddiejaoude)
```
Comment (Panquesito7) [inline docs/_layout/navbar.md:4]:
```suggestion
- [YouTube](https://youtube.com/c/eddiejaoude)
```

File: docs/_layout/sidebar.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
   - [How to Use GitHub](/tips/githubtips.md)
   - [Common Terms](/tips/glossary.md)
   - [Open Source Resources](/tips/OpenSourceResources.md)
   - [Open Source Tips](/tips/OpenSourceTips.md)
-- YouTube
-  - [Generic Description Template](/youtube/generic.md)
-  - [LiveStream Reviews Template](/youtube/livestream.md)
+
+- Templates
+  - [Youtube Generic Template](/youtube/generic.md)
```
Comment (Panquesito7) [inline docs/_layout/sidebar.md:17]:
```suggestion
  - [YouTube Generic Template](/youtube/generic.md)
```

File: docs/index.html [hunk 3]
```diff
@@ -1,66 +1,63 @@
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/docsify/4.12.1/docsify.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/docsify-darklight-theme@latest/dist/index.min.js" type="text/javascript">
+</script>
+<script src="https://cdn.jsdelivr.net/npm/docsify-plugin-flexible-alerts@1"></script>
+<script src="https://unpkg.com/docsify-copy-code@2"></script>
+<!-- Prism (syntax highlighting) -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.25.0/components/prism-typescript.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.25.0/components/prism-markdown.min.js"></script>
+
+</html>
```
Comment (Panquesito7) [inline docs/index.html:63]: Missing an endline.
```suggestion
</html>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
The existing theme specifically used colours indicated in the branding repository. The new theme should do the same.

**Comment by krishguptadev** [comment]
Okey! I'll work on that.

**Review by Vyvy-vi — COMMENTED** [review]
looks good, though the navbar is kinda unnecessary and leads to too much noise in the top right corner - 
Screenshot 2021-12-07 at 10 08 47 PM

File: docs/index.html [hunk 4]
```diff
@@ -1,66 +1,106 @@
...
```
Comment (Panquesito7) [inline docs/index.html:106]: I think this needs an endline? 😅
```suggestion
</html>

```

**Comment by krishguptadev** [comment]
> looks good, though the navbar is kinda unnecessary and leads to too much noise in the top right corner - Screenshot 2021-12-07 at 10 08 47 PM

I am not sure, but IMHO the links to discord and sutff should not be in sidebar that is why I added the here

File: docs/index.html [hunk 5]
```diff
@@ -1,66 +1,106 @@
...
+<body>
+  <div id="app">
+    <h1>EddieHub Support Repo</h1>
+    <p>Resources to help our community grow and thrive.</p>
+  </div>
+</body>
+<script>
+  window.$docsify = {
+    el: '#app',
+    repo: "eddiehubcommunity/support",
```
Comment (Panquesito7) [inline docs/index.html:27]:
```suggestion
    repo: "EddieHubCommunity/support",
```

**Comment by Panquesito7** [comment]
We could also look into adding a favicon. It would look better. 🙂

**Comment by krishguptadev** [comment]
> We could also look into adding a favicon. It would look better. 🙂

Not the purpose of the PR. Will add it in another one though 🙂

**Comment by ezeamaka2** [comment]
Been trying to create a pull request, so I can request access to join this amazing community. Please someone should add me

**Comment by krishguptadev** [comment]
> Been trying to create a pull request, so I can request access to join this amazing community. Please someone should add me

You need to create an Issue: https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization

**Comment by ezeamaka2** [comment]
> > Been trying to create a pull request, so I can request access to join this amazing community. Please someone should add me
> 
> You need to create an Issue: https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization

Oh, thanks

**Review by nhcarrigan — APPROVED** [review]
Let's go ahead and get this in.

**Pull request merged**